arm/Microsoft.KeyVault/vaults/parameters/parameters.json

```diff
@@ -25,40 +25,40 @@
         ]
       }
     },
-    "accessPolicies": {
```

Comment:
Let's remove that. It's described in the ReadMe.

We should probably do the same for every case where we have examples for 'roleAssignments'